Common/Utilities.cs

```diff
@@ -1,53 +1,46 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using Azure.Messaging.ServiceBus;
+using Azure.Storage.Blobs;

```

Comment:
After looking end-to-end, this class seems to be copy/pasted and includes way, way more than is needed for this sample.  Our goal is to be clear and approachable, focused on the task at hand to ensure that developers can follow and apply patterns to their own applications.  This class seems about as far from that as you can get; it needs to be burned with fire.

Reply:
I think we could keep this Utilities.cs, but only keeps serveral program related method. The reason is that there are some other PRs [example1](https://github.com/Azure-Samples/service-bus-dotnet-manage-publish-subscribe-with-advanced-features/pull/2), [example2](https://github.com/Azure-Samples/service-bus-dotnet-manage-publish-subscribe-with-basic-features/pull/2), [example3](https://github.com/Azure-Samples/service-bus-dotnet-manage-queue-with-advanced-features/pull/4), [example4](https://github.com/Azure-Samples/service-bus-dotnet-manage-queue-with-basic-features/pull/3) using such method i kept in Utilities.cs . The origin purpose i believe is avoid redundancy. I will work with these examples after this is merged.